pandas/core/numba_/executor.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
```

Comment:
pandas/core/_numba maybe or pandas/core/_engines/_numba_

just a thought

Reply:
IOW i think should make private by default

Reply:
Sure. Even thought `pandas.core` is "private", I can change the path.

Reply:
yeah even though :->